lib/experiments-page.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Bootstraping the Gutenberg experiments page.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * The main entry point for the Gutenberg experiments page.
+ *
+ * @since 5.2.3
+ *
+ * @param string $page The page name the function is being called for, `'gutenberg_customizer'` for the Customizer.
+ */
+function the_gutenberg_experiments( $page = 'gutenberg_page_gutenberg-experiments' ) {
+	?>
+	<div
+		id="experiments-editor"
+		class="blocks-experiments-container"
+	>
+	<?php settings_errors(); ?>
+	<form method="post" action="options.php">
+		<?php settings_fields( 'gutenberg-experiments' ); ?>
+		<?php do_settings_sections( 'gutenberg-experiments' ); ?>           
+		<?php submit_button(); ?>
+	</form>
+	</div>
+	<?php
+}
+
+/**
+ * Set up the experiments settings.
+ *
+ * @since 5.2.3
+ */
+function initialize_experiments_settings() {
+	add_settings_section(
+		'gutenberg_experiments_section',
+		'Experiment settings',
```

Comment:
We need a i18n function.

Reply:
@tellthemachines an i18n function, meaning to make the string use translations. suhc as:
```
__('Experiment settings' ),
```